<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Penggambaran Trilaterasi</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="top_label">
     <property name="text">
      <string>Input untuk penggambaran trilaterasi</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="trilaterasi_titik_1">
       <property name="text">
        <string>Titik 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="trilaterasi_koord_1">
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Koordinat Titik 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_jarak_1">
       <property name="toolTip">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Jarak dari titik 1</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>23</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="trilaterasi_titik_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Titik 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="trilaterasi_koord_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Koordinat Titik 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_jarak_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Jarak dari titik 2</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>23</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="trilaterasi_titik_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Titik 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="trilaterasi_koord_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Koordinat Titik 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="input_jarak_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Jarak dari titik 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_titik_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="trilaterasi_ok">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="trilaterasi_cancel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
